test/reorder/amd.cpp

```diff
@@ -40,7 +40,8 @@ class Amd : public CommonTestFixture {
     std::shared_ptr<matrix_type> dmtx;
 };
 
-TYPED_TEST_SUITE(Amd, gko::test::ValueIndexTypes, PairTypenameNameGenerator);
+TYPED_TEST_SUITE(Amd, gko::test::ValueIndexTypesBase,
+                 PairTypenameNameGenerator);
```

Comment:
Necessary?

Reply:
good catch! It can pass with half. It was missed in the reorder commit

Reply:
I also change it for core/test/reorder/amd